lerna.json

```diff
@@ -1,7 +1,7 @@
 {
   "npmClient": "yarn",
   "useWorkspaces": true,
-  "version": "13.0.0",
+  "version": "14.0.0-preview.0",
```

Comment:
I would suggest `next` as tag and version name

Reply:
I think it makes sense to have a version here as we need that when we deploy the demo app. How about `14.0.0-nightly` or just  `14.0.0-master`?  `14.0.0-next` would also be fine.

Reply:
That's what I meant. I would just suggest to take 'next' instead of 'preview' because I think that's more common.

Reply:
The tag name is already `next`: https://github.com/xing/hops/blob/master/lerna.json#L14 and I changed preview to nightly.